src/test/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilderTest.java

```diff
@@ -156,4 +157,24 @@ void testBillingModeIsPayPerRequestForDefaultCreateTableRequestsWithPayPerReques
                 .collect(Collectors.toList());
     }
 
+    @Test
+    void testTableBuilderInterface() {
+        // Weak cast only for TableBuilder testing purposes
+        TableBuilder tableBuilder = SharedTableBuilder.builder()
+            .withAmazonDynamoDb(LOCAL_DYNAMO_DB)
+            .withTablePrefix(tablePrefix)
+            .withPrecreateTables(true)
+            .withContext(MT_CONTEXT);
+
+        tableBuilder.withBillingMode(BillingMode.PAY_PER_REQUEST);
+
+        ((SharedTableBuilder) tableBuilder).build();
+
+        for (String table: getPrefixedTables()) {
```

Comment:
space before the colon

Reply:
fixed in 9bf98d3f8d5d83df92a8a9608246ef61c9ad4772